openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -527,10 +527,29 @@ def run(self, *, dry=False, verbose=True,
         )
 
         #  b. Write out a PDB containing the subsampled hybrid state
+        A_atoms = {hybrid_factory.old_to_hybrid_atom_map[i]
```

Comment:
@IAlibay Is it worth adding this to HTF? Or is it already there and I missed it

Reply:
I think everything you need should already be there under: https://github.com/OpenFreeEnergy/openfe/blob/987009e607dce25b2b1d6dce20d2b50cba01c6d0/openfe/protocols/openmm_rfe/_rfe_utils/relative.py#L439-L442

So you'd be looking for `_atom_classes['unique_new_atoms']` and `_atom_classes['unique_old_atoms']` and `_atom_classes['core_atoms']`